app/models/concerns/indexable.rb

```diff
@@ -1,14 +1,14 @@
 module Indexable
   extend ActiveSupport::Concern
 
-  INDEX_NAME = [ENV["ALGOLIA_INDEX_PREFIX"], DOMAIN, Vacancy].compact.join("-").freeze
+  INDEX_NAME = [Rails.env.test? ? nil : ENV.fetch("ALGOLIA_INDEX_PREFIX", nil), DOMAIN, Vacancy].compact.join("-").freeze
```

Comment:
This has been here for a while but at least the first bit feels like it should live somewhere in configuration instead rather than us testing whether we are in a specific environment deep within app code 😕 

How about a `config.algolia_index_prefix = ENV.fetch("ALGOLIA_INDEX_PREFIX", nil)` in `config/application.rb`, and a `config.algolia_index_prefix = nil` in `config/enviroments/test.rb`?

Reply:
Yeah that sounds good to me